makerspace_backend/runserver.py

```diff
@@ -0,0 +1,2 @@
+from makerspace_backend import app
+app.run(debug=True)
```

Comment:
You sure we want to make it debug true by default? And do we have all the dependencies here?